.github/workflows/release-please.yml

```diff
@@ -35,55 +37,94 @@ jobs:
       hashes: ${{ steps.full-release.outputs.hashes }}
     steps:
       - uses: actions/checkout@v4
+
+      - uses: ./.github/actions/setup-env
```

Comment:
Maybe leftover?

Reply:
Or rather I don't see this action anymore.

Reply:
Still fixing the source env code, but will fix this guy.  Good catch.

Reply:
Done